frontend/components/events/create/forms/PreviewForm.tsx

```diff
@@ -14,67 +14,64 @@ type PreviewFormProps = BasicData & {
   updateField: (fields: Partial<FormData>) => void;
 };
 
-export const PreviewForm = ({
-  form,
-  user,
-  imagePreviewUrl,
-  updateField,
-}: PreviewFormProps) => {
+export const PreviewForm = ({ form, user, imagePreviewUrl, updateField }: PreviewFormProps) => {
   const dateString = form.date + " " + form.startTime;
   var [datePart, timePart] = dateString.split(" ");
   var [year, month, day] = datePart.split("-");
   var [hours, minutes] = timePart.split(":");
-  var myDate = new Date(
-    parseInt(year),
-    parseInt(month) - 1,
-    parseInt(day),
-    parseInt(hours),
-    parseInt(minutes)
-  );
+  var myDate = new Date(parseInt(year), parseInt(month) - 1, parseInt(day), parseInt(hours), parseInt(minutes));
 
   return (
-    <div className="md:grid md:grid-cols-2 mt-4 items-start">
-      <div className="flex flex-col items-center mb-12 md:mb-0 md:mt-16">
-        <div className="flex flex-col">
-          <div className="font-bold">Name of event:</div>
-          {form.name}
-        </div>
-        <div className="flex flex-col">
-          <div className="font-bold">Location of event:</div>
-          {form.location}
-        </div>
-        <div className="flex flex-col">
-          <div className="font-bold">Start Time of event:</div>
-          {form.startTime}
+    <div className="md:grid md:grid-cols-3 gap-6 mt-20 items-start mb-8">
+      <div className="col-span-1 mt-6">
+        <div className="text-xl lg:text-xl font-semibold mb-2 border-b-2 border-gray-300 pb-1">Name</div>
+        <p className="text-m">{form.name}</p>
+
+        <div className="text-xl lg:text-xl font-semibold mt-6 mb-2 border-b-2 border-gray-300 pb-1">Location</div>
+        <p className="text-m">{form.location}</p>
+
+        <div className="text-xl lg:text-xl font-semibold mt-6 mb-2 border-b-2 border-gray-300 pb-1">
```

Comment:
we want the date as well! not just the time

Reply:
yessir added in